registrations/models.py

```diff
@@ -705,6 +708,43 @@ class SignUpGroup(
         {"name": "contact_person"},
     )
 
+    @transaction.atomic
+    def delete(self, *args, **kwargs):
+        contact_person = getattr(self, "contact_person", None)
+        is_attending = True if self.attending_signups else False
```

Comment:
You can also do:
```suggestion
        is_attending = bool(self.attending_signups)
```

Reply:
Yes indeed, this was the solution I wanted yesterday afternoon. Now changed as suggested.